frontend/package.json

```diff
@@ -322,7 +326,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged && npm run lint-stylelint"
```

Comment:
I haven't used `lint-stylelint-fix` here because this runs on the commit command after we've added the files to index. It would change files just before the actual commit happens and as a result the older added files will be the ones committed, not the auto-fixed ones, thus making the branch "dirty" and requiring a manual add and amend. Hence I've used `lint-stylelint` that fails if an error is detected. Folks can still use `lint-stylelint-fix` in such a case to auto-fix all auto-fixable errors.

Also, please take a look [here](https://github.com/openshift/console/pull/8360#issuecomment-811701231) to get an idea about how it's going to look. Thanks!